Pipfile

```diff
@@ -40,6 +40,7 @@ ciso8601 = ">=2.1"
 pandas = ">=0.25"
 sentry-sdk = ">=0.13"
 ujson = ">=1.35"
+responses = "*"
```

Comment:
does this need to be in both the `packages` and `dev-packages` list? Looks like its only used in a test, so maybe this can be removed.

Reply:
Whoops! I'll fix.